src/server/pfs/cmds/cmds.go

```diff
@@ -1311,6 +1312,36 @@ Objects are a low-level resource and should not be accessed directly by most use
 	fsck.Flags().BoolVarP(&fix, "fix", "f", false, "Attempt to fix as many issues as possible.")
 	commands = append(commands, cmdutil.CreateAlias(fsck, "fsck"))
 
+	var seed int64
+	runLoadTest := &cobra.Command{
+		Use:     "{{alias}} <spec>",
+		Short:   "Run a PFS load test.",
+		Long:    "Run a PFS load test.",
+		Example: pfsload.LoadSpecification,
+		Run: cmdutil.RunFixedArgs(1, func(args []string) (retErr error) {
+			c, err := client.NewOnUserMachine("user")
+			if err != nil {
+				return err
+			}
+			defer func() {
+				if err := c.Close(); retErr == nil {
+					retErr = err
+				}
+			}()
+			spec, err := ioutil.ReadFile(args[0])
```

Comment:
Is `len(args) > 0` ?  If the users can touch this then it shouldn't be able to panic.

Reply:
The wrapping function (`RunFixedArgs`) checks that exactly one argument is provided, and errors otherwise.